<launch>
  <node pkg="hebi_cpp_api_examples" type="arm_node_action" name="arm_node_action" />
</launch>
